index.js

```diff
@@ -0,0 +1,65 @@
+import { createReloadable } from '@artsy/express-reloadable';
+import CopyWebpackPlugin from 'copy-webpack-plugin';
+import FriendlyErrorsWebpackPlugin from 'friendly-errors-webpack-plugin';
+import express from 'express';
+import graphQLHTTP from 'express-graphql';
+import path from 'path';
+import webpack from 'webpack';
+import webpackMiddleware from 'webpack-dev-middleware';
+
+import schema from './src/data/schema';
+
+const PORT = 8080;
+
+const app = express();
+
+app.use('/graphql', graphQLHTTP({ schema }));
+
+const mountAndReload = createReloadable(app, require);
+mountAndReload(path.resolve(__dirname, './src'));
+
+const webpackConfig = {
+  mode: 'development',
+
+  entry: ['babel-polyfill', './src/client'],
+
+  output: {
+    path: '/',
+    filename: 'bundle.js',
+  },
+
+  module: {
+    rules: [
+      // See: https://github.com/aws/aws-amplify/issues/686#issuecomment-387710340
```

Comment:
i'm surprised this is necessary... graphql.js shouldn't be getting pulled into the browser bundle, should it?

Reply:
This is due to `react-relay-network-modern-ssr` publishing `.mjs` files. I haven't had the time to look into this further, but this temporarily fixes it.